src/mame/layout/enmirage.lay

```diff
@@ -0,0 +1,210 @@
+		<element name="button_Value" ref="button_grey" inputtag="pb7" inputmask="0x40"><bounds x="149" y="232" width="44" height="24" /></element>
+		<element name="button_7" ref="button_grey" inputtag="pb7" inputmask="0x04"><bounds x="282" y="232" width="44" height="24" /></element>
+		<element name="button_8" ref="button_grey" inputtag="pb6" inputmask="0x10"><bounds x="375" y="232" width="44" height="24" /></element>
+		<element name="button_9" ref="button_grey" inputtag="pb6" inputmask="0x04"><bounds x="468" y="232" width="44" height="24" /></element>
+		<element name="button_Load" ref="button_grey" inputtag="pb5" inputmask="0x10"><bounds x="597" y="232" width="44" height="24" /></element>
+
+		<!-- Buttons, row 4 -->
+		<element name="button_Off/Down" ref="button_grey" inputtag="pb7" inputmask="0x20"><bounds x="38" y="325" width="44" height="24" /></element>
+		<element name="button_On/Up" ref="button_grey" inputtag="pb7" inputmask="0x08"><bounds x="149" y="325" width="44" height="24" /></element>
+		<element name="button_Cancel" ref="button_grey" inputtag="pb7" inputmask="0x80"><bounds x="282" y="325" width="44" height="24" /></element>
+		<element name="button_0/Prog" ref="button_white" inputtag="pb6" inputmask="0x20"><bounds x="375" y="325" width="44" height="24" /></element>
+		<element name="button_Enter" ref="button_grey" inputtag="pb6" inputmask="0x80"><bounds x="468" y="325" width="44" height="24" /></element>
+		<element name="button_Save" ref="button_grey" inputtag="pb5" inputmask="0x20"><bounds x="597" y="325" width="44" height="24" /></element>
+
+		<!-- Lines -->
+		<element name="line_1" ref="button_white"><bounds x="281" y="82" width="659" height="6" /></element>
+		<element name="line_2" ref="button_white"><bounds x="281" y="175" width="659" height="6" /></element>
+		<element name="line_3" ref="button_white"><bounds x="34" y="268" width="649" height="6" /></element>
+		<element name="line_4" ref="button_white"><bounds x="0" y="361" width="939" height="6" /></element>11
+		<element name="line_5" ref="button_white"><bounds x="38" y="367" width="155" height="6" /></element>
+		<element name="line_6" ref="button_white"><bounds x="282" y="367" width="230" height="6" /></element>
+		<element name="line_7" ref="button_white"><bounds x="597" y="367" width="44" height="6" /></element>
+		<element name="line_8" ref="button_white"><bounds x="711" y="367" width="71" height="6" /></element>
+		<element name="line_9" ref="button_white"><bounds x="895" y="367" width="44" height="6" /></element>
+		<element name="line_10" ref="button_white"><bounds x="60" y="149" width="3" height="29" /></element>
+		<element name="line_11" ref="button_white"><bounds x="123" y="149" width="3" height="29" /></element>
+		<element name="line_12" ref="button_white"><bounds x="38" y="175" width="44" height="3" /></element>
+		<element name="line_13" ref="button_white"><bounds x="123" y="175" width="75" height="3" /></element>
```

Comment:
Are you really trying to attach all these to outputs or make them interactive in some way?  The `name` attribute attaches elements to outputs.